src/relayclient/ContractInteractor.ts

```diff
@@ -221,6 +221,22 @@ export default class ContractInteractor {
     return contract.getPastEvents('allEvents', Object.assign({}, options, { topics }))
   }
 
+  async getPastLogs (eventName: EventName, options: PastEventOptions): Promise<Log[]> {
```

Comment:
good start, but:
- no test
- not much of use without parsing params.

Reply:
Added to backlog